Update README_EN.md (#1)
plusls/oh-my-minecraft-client: README_EN.md

> Make Minecraft Client Great Again!

Reviewer: ```suggestion
Make Your Minecraft Client Great Again!
```

## Description
Oh My Minecraft Client is a client-side mod, which helps you manage your hands. The mod is written for the Fabric Loader.

> ## NOTE: My Minecraft Client also requires the MaLiLib library mod by masady:

Reviewer: ```suggestion
## NOTE: Oh My Minecraft Client also requires the MaLiLib library mod by masady:
```

https://www.curseforge.com/minecraft/mc-mods/malilib

The default hotkey to open the in-game config GUI is O + C

## Features

### Disable Break Scaffolding

> You can break the scaffolding with the items in **breakScaffoldingWhiteList**.

Reviewer: ```suggestion
You can only break scaffolding with the items in **breakScaffoldingWhiteList**.
```

### Disable Move Down In Scaffolding

You can move down in scaffolding when the item of **moveDownInScaffoldingWhiteList** in your hand.

> ### Forced Break Cooling

Reviewer: ```suggestion
### Forced Break Cooldown
```

> Players will have a 5GT cooldown after breaking the block instantaneously, which will prevent you from accidentally destroying components with the Haste 2 effect.

Reviewer: ```suggestion
Players will have a 5gt cooldown after breaking the block instantaneously, which will prevent you from accidentally destroying components with the Haste 2 effect.
```

Reviewer: 5gt is written lowercase universally in technical minecraft, from what I have seen.

### Highlight Lava Sources

> Any lava source will be highlighted with a special texture.

Reviewer: ```suggestion
Lava sources will be highlighted with a special texture.
```

It looks like:

![highlightLavaSourceOff](./docs/img/highlightLavaSourceOff.png)

![highlightLavaSourceOn](./docs/img/highlightLavaSourceOn.png)

> ### Wandering Trader Glow

Reviewer: ```suggestion
### Glowing Wandering Trader
```

> The Wandering Trader will have the glowing affect, so it in Invisibility.

Reviewer: ```suggestion
The Wandering Trader will have the glowing effect, so you can see where it is when it is invisible.
```

![highlightLavaSourceOn](./docs/img/highlightWanderingTrader.png)

### Bed Explosion Prevention

> You can prevent breaking beds in The Nether or The End. (Not included with the respawn anchor in Overworld).

Reviewer: ```suggestion
You can prevent breaking beds in The Nether or The End. (Breaking the respawn anchor in The Overworld is not included).
```

### Real Sneaking

> Players cannot ascend or descend when sneaking.

Reviewer: ```suggestion
You cannot ascend or descend non-full blocks when sneaking (e.g. carpets).
```
